DeepCat.js

```diff
@@ -49,10 +73,37 @@
 			}
 		} );
 
-		refreshSearchTermMock();
+		if(refreshSearchTermMock())
+			showHint();
 		checkErrorMessage();
 	} );
 
+	function showHint() {
+		if( !mw.cookie.get( makeHintboxCookieToken(mw.msg('hintbox-text'))) ) {
+			var parent= document.getElementById('mw-content-text');
+			var sresults= document.getElementsByClassName('searchresults')[0];
+			var d= parent.insertBefore(document.createElement('div'), sresults);
+			d.style.marginTop= "1em";
+			d.style.marginBottom= "1em";
+			d.innerHTML=
```

Comment:
This is a bit dangerous because it may not work in all browsers. Better use jQuery's html function.